cli/tests/unit/test_marathon_validation.py

```diff
@@ -0,0 +1,62 @@
+import collections
+import json
+
+import pkg_resources
+from dcos import util
+from dcoscli.marathon import main
+
+import pytest
+
+ResourceData = collections.namedtuple(
+    'ResourceData',
+    ['properties', 'expected'])
+
+clean_response = json.loads(
+    pkg_resources.resource_string(
+        'tests',
+        'data/marathon/apps/cleaned_response.json').decode('utf-8'))
```

Comment:
I would just turn this into a method. I am okay reading that file multiple times in the tests.